static/js/components/CoverImageInput.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+import React from "react"
+import Dropzone from "react-dropzone"
+
+import { editorUpdateFormShim } from "./Editor"
+
+type Props = {
+  image: ?File | string,
+  onUpdate: Function,
+  setPhotoError?: Function
+}
+
+export default class CoverImageInput extends React.Component<Props> {
+  setCoverImage = (image: ?File) => {
+    const { onUpdate } = this.props
+    editorUpdateFormShim("cover_image", onUpdate)(image)
+  }
+
+  handleImageDrop = (images: Array<File>) => {
+    const [image] = images
+    this.setCoverImage(image)
+  }
+
+  clearCoverImage = () => {
+    this.setCoverImage(undefined)
+  }
+
+  render() {
+    const { image, setPhotoError } = this.props
+
+    return (
+      <div className="article-banner-image-wrapper">
+        <div className="article-banner-image">
+          {image ? (
+            <div className="cover-img-container">
+              <img
+                src={
+                  typeof image === "string" || image instanceof String
+                    ? image
+                    : URL.createObjectURL(image)
+                }
+              />
+              <button
+                className="high-contrast-outlined"
+                type="button"
+                onClick={this.clearCoverImage}
+              >
+                remove image
+              </button>
+            </div>
+          ) : (
+            <Dropzone
+              onDrop={this.handleImageDrop}
+              className="photo-upload-dialog photo-active-item photo-dropzone dashed-border"
+              style={{ height: 150 }}
```

Comment:
Any reason this shouldn't just be set in CSS?

Reply:
I'll try, I just copy-pasted from our other usage of the `Dropzone` component

Reply:
haha, not sure why we were setting this like this before! works fine to set via CSS.